### PR TITLE
[Fix](compile) Fix control reaches end of non-void function

### PR DESCRIPTION
**Pull request by bobhan1**

### CODE REVIEW

**Review by dataroaring — APPROVED** [review]
LGTM

**Comment by dataroaring** [comment]
run buildall

**Review by BiteTheDDDDt — APPROVED** [review]

File: be/src/olap/partial_update_info.h [hunk 1]
```diff
@@ -66,6 +66,7 @@ struct PartialUpdateInfo {
         case UniqueKeyUpdateModePB::UPDATE_FLEXIBLE_COLUMNS:
             return "flexible partial update";
         }
+        return "";
```
Comment (zhannngchen) [inline be/src/olap/partial_update_info.h:69]:
```suggestion
switch () {
case xx:
    // do sth.
case yy:
    // do sth.
default:
        return "";
}
```

Comment (zhannngchen) [reply be/src/olap/partial_update_info.h:69]: see Google C++ style : https://google.github.io/styleguide/cppguide.html#Switch_Statements

**Comment by bobhan1** [comment]
run cloud_p0

**Pull request merged**